### PR TITLE
PR - 2 - Fixed testing issues and updated readmes

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/graph-appcatalog-lifecycle/nodejs/index.js [hunk 1]
```diff
@@ -51,7 +61,7 @@ adapter.onTurnError = async (context, error) => {
 // Define the state store for your bot.
 // See https://aka.ms/about-bot-state to learn more about using MemoryStorage.
 // A bot requires a state storage system to persist the dialog and user state between messages.
-const memoryStorage = new MemoryStorage();
+//const memoryStorage = new MemoryStorage();
```
Comment (Prithvi-MSFT) [inline samples/graph-appcatalog-lifecycle/nodejs/index.js:64]: Please remove commented code

File: samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ class Dashboard extends Component {
 
     // Exchange client token with server token and fetch the pinned message details.
     exchangeClientTokenForServerToken = async (token) => {
-        var id = "12345";
+        var id = "ChatId";  //Replace {{ChatId ex - 19:75e4d14a2c2c4644a7adaf4e3976d17b@thread.v2}}
```
Comment (Prithvi-MSFT) [inline samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx:88]: chat id needs to be static or dynamic value ?

File: samples/graph-appcatalog-lifecycle/nodejs/index.js (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-appcatalog-lifecycle/nodejs/index.js:64]: Fixed

File: samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx:88]: Replace the placeholder 'chatid' with the actual chat ID.

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8060232197**
image

File: samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx:88]: image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

File: samples/app-anonymous-users/nodejs/ClientApp/src/components/facebook-auth-end.jsx [hunk 3]
```diff
@@ -20,17 +20,16 @@ const FacebookAuthEnd = props => {
...
                 // Use localStorage to avoid passing the token via notifySuccess; instead we send the item key.
-                let key = "auth.result";
-                localStorage.setItem(key, JSON.stringify({
-                    idToken: code,
-                }));
+                
+                //let key = "auth.result";
+                //localStorage.setItem(key, JSON.stringify({
+                    //idToken: code,
+                //}));
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/nodejs/ClientApp/src/components/facebook-auth-end.jsx:30]: Fixed

File: samples/app-anonymous-users/nodejs/ClientApp/src/components/shareview.jsx [hunk 4]
```diff
@@ -129,20 +129,19 @@ const ShareView = () => {
...
                 width: 600,
                 height: 535
             })
             .then((result) => {
-                let data = localStorage.getItem(result);
-                let tokenDetails = JSON.parse(data);
-                localStorage.removeItem(result);
-                resolve(tokenDetails);
+                //let data = localStorage.getItem(result);
+                //let tokenDetails = JSON.parse(data);
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/nodejs/ClientApp/src/components/shareview.jsx:142]: Fixed

File: samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/graph-pinned-messages/nodejs/ClientApp/src/components/dashboard.jsx:88]: This demo tenant ChatId - 
image

File: samples/graph-pinned-messages/nodejs/controller/pin-message-controller.js [hunk 5]
```diff
@@ -13,6 +13,10 @@ const getGraphAccessToken = async (req, res) => {
     var chatId = req.query.chatId;
     var scopes = ["https://graph.microsoft.com/User.Read"];
 
+    console.log("token_" + token);
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/graph-pinned-messages/nodejs/controller/pin-message-controller.js:16]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8136527988
image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Pull request merged**